usr/share/rear/layout/save/FDRUPSTREAM/Linux-s390/990_copy_disklayout_file.sh

```diff
@@ -23,8 +23,17 @@ opath=$( output_path $scheme $path )
 
 if [[ "$ZVM_NAMING" == "Y" && "$ARCH" == "Linux-s390" ]] ; then 
       VM_UID=$(vmcp q userid |awk '{ print $1 }')
-      LogPrint "s390 dislayout.conf will be saved as $(opath)/$VM_UID.disklayout.conf"
-      mkdir -p ${opath}/"$VM_UID"
-      cp $v $DISKLAYOUT_FILE ${opath}/"$VM_UID".disklayout.conf || Error "Failed to copy disklayout.conf ($DISKLAYOUT_FILE) to ${opath}/"$VM_UID".disklayout.conf"
+      if [[ -z $VM_UID ]] ; then
+            LogPrint "VM UID is not set, VM UID is set from call to vmcp.  Please make sure vmcp is available and 'vmcp q userid' returns the vm login id"
+            LogPrint "VM UID for disklayout.conf will be set to 'DEFAULT'"
+            VM_UID="DEFAULT"
+      fi
+      if [[ -z $opath ]] ; then
+            LogPrint "Warning: output path is not set, please check OUTPUT_URL in local.conf."
+      fi
+
+      LogPrint "s390 disklayout.conf will be saved as $opath/$VM_UID.disklayout.conf"
+      mkdir -pv $opath
+      cp $v $DISKLAYOUT_FILE $opath/$VM_UID.disklayout.conf || Error "Failed to copy disklayout.conf ($DISKLAYOUT_FILE) to opath/$VM_UID.disklayout.conf"
```

Comment:
`opath` should be `${opath} in the Error message.

Reply:
Fixed via
https://github.com/rear/rear/commit/7bcad6b517aad2fcaf42d2a12464b42b5508a856
plus much more code cleanup that "just happened by the way" as I noticed it ;-)